### PR TITLE
Move macOS CI to Github Actions

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by alex** [comment]
I'm baffled. `==> Python coveragepy not found` even though there's a `pip install tox coverage`...

**Comment by alex** [comment]
I believe this is blocked on https://github.com/codecov/codecov-action/pull/67

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -51,7 +88,7 @@ jobs:
...
           TOXENV: ${{ matrix.PYTHON.TOXENV }}
 
-      - uses: codecov/codecov-action@v1
-        with:
-          name: "Python ${{ matrix.PYTHON.VERSION }} on ${{ matrix.WINDOWS.WINDOWS }}"
-          fail_ci_if_error: true
+      - name: Upload coverage
+        run: |
+          curl -o codecov.sh -f https://codecov.io/bash
+          bash codecov.sh -n "Python ${{ matrix.PYTHON.VERSION }} on ${{ matrix.WINDOWS.WINDOWS }}" -Z
```
Comment (reaperhulk) [inline .github/workflows/ci.yml]: Looks like this carried over from the windows section

Comment (alex) [reply .github/workflows/ci.yml]: Yes, I did it intentionaly. Consistency!

**Pull request merged**